docs/data/material/varnish/varnish1-to-varnish2-migration/ShapeExample.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+
+export default function Demo() {
+  const theme = useTheme();
+  return (
+    <div
+      style={{ width: '100%', height: '50px', backgroundColor: theme.color2.B5.hex }}
+    >
+      <div
+        style={{
```

Comment:
please make a styled component or use sx, we dont want to encourage inline styles (people will copy paste this)